README.md

```diff
@@ -39,6 +39,27 @@ Can be sourced from jcenter like so:
       <classifier>sources|tests|javadoc|all</classifier> (Optional)
     </dependency>
 	
+**Note**: You need to have bintray maven repository configured to be able to pull bitbucket-rest dependency.
+
+### Maven repository Configuration:
+
+* Download a sample settings.xml file that has configured profile to pull from the correct maven repository using this link: [settings.xml](https://bintray.com/repo/downloadMavenRepoSettingsFile/downloadSettings?repoPath=%2Fcdancy%2Fjava-libraries)
+
+**OR**
+
+* Add the following repository to the project pom.xml file under project tag:
+```
+    <repositories>
+      <repository>
+        <snapshots>
+          <enabled>false</enabled>
+        </snapshots>
+        <id>bintray-cdancy-java-libraries</id>
+        <name>bintray</name>
+        <url>https://dl.bintray.com/cdancy/java-libraries</url>
+      </repository>
+    </repositories>
+```
```

Comment:
Would it be better to just do something like the below and link to it?

https://stackoverflow.com/questions/44265547/how-to-properly-specify-jcenter-repository-in-maven-config

Reply:
@cdancy 
I can do this to the embeded one "second option"
But for the first option, settings.xml, I find it easier to provide the link and download the settings.xml instead of adding the text and making README very big.
I can check if jcenter is providing a link to a sample settings.xml, or else, we can add it in the repo and link to it
What do you think?

Reply:
@cdancy OK, I find a link to settings.xml for jcenter file, modified both options to use jcenter